ROADMAP.md

```diff
@@ -2,30 +2,47 @@
 
 ## Future priorities
 
-In rough order of priority:
-
-* **Support Encrypted Client Hello (Client Side)**.
-  Encrypted Client Hello is an upcoming standard from the TLS WG providing better
-  protection for some of the data sent by a client in the initial Client Hello
-  message.
-  rustls/rustls#1718
-
-* **Additional Performance Optimization**.
-  Additional performance optimization including CPU usage, latency, and memory
-  usage. The goal is to outperform OpenSSL across the board if we are not already.
+Specific features, in rough order of priority:
 
 * **Support RFC 8879 Certificate Compression**.
   Support for a TLS extension that substantially shrinks certificates (one of the
   largest parts of the TLS handshake), improving handshake latency by decreasing
   bandwidth used.
   rustls/rustls#534
 
+* **Support Encrypted Client Hello (Client Side)**.
+  Encrypted Client Hello is an upcoming standard from the TLS WG providing better
+  protection for some of the data sent by a client in the initial Client Hello
+  message.
+  rustls/rustls#1718
+
 * **Enforce Confidentiality / Integrity Limits**.
   The QUIC use of TLS mandates limited usage of AEAD keys. While TLS 1.3 and 1.2
   do not require this, the same kinds of issues can apply here, and we should
   consider implementing limits for TLS over TCP as well.
   rustls/rustls#755
 
+* **Address asynchronous handshake interruption**.
+  Allow completion of user-provided operations to be deferred.
+  rustls/rustls#850
+
+* **Support Encrypted Client Hello (ECH) (Server Side)**.
+  Encrypted Client Hello is an upcoming standard from the TLS WG providing better
+  protection for some of the data sent by a client in the initial Client Hello
+  message. Rustls already supports client side ECH, we will add server side support.
+
```

Comment:
link to https://github.com/rustls/rustls/issues/199 ?

Reply:
On second thought: a more specific issue is better, https://github.com/rustls/rustls/issues/1980